### PR TITLE
Added more functionalities of multipath [Waiting for util]

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Added new functions like removing and adding of individual and
mpath. suspending and resuming of mpaths.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[multipath_suspend_resume_functions_log_v2.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3646427/multipath_suspend_resume_functions_log_v2.txt)

**Comment by Naresh-ibm** [comment]
there are few library functions of multipath yet to merge and this test uses them.
So just review as of now and you can merge once the library functions get merge.

library function patch is here https://github.com/avocado-framework/avocado/pull/3313

**Comment by narasimhan-v** [comment]
@Naresh-ibm any updates on this ?

File: io/disk/multipath_test.py [hunk 1]
```diff
@@ -185,9 +185,41 @@ def test(self):
             for path in path_dic['paths']:
                 if multipath.reinstate_path(path) is False:
                     msg += "%s did not recovered  in all path fail\n" % path
+
+            # Removing all paths for short time and adding back
+            self.log.info("Removing all paths and adding back")
+            for path in path_dic['paths']:
+                if multipath.remove_path(path) is False:
```
Comment (abdhaleegit) [inline io/disk/multipath_test.py]: @Naresh-ibm the return is a boolean and you can easily do `if not multipath.remove_path(path):` instead of `is false`:

**Comment by PraveenPenguin** [comment]
> @Naresh-ibm any updates on this ?

@Naresh-ibm can you please add self-test also for multipath lib as it is good to go in

**Comment by Naresh-ibm** [comment]
@PraveenPenguin  sure Praveen, will resume on this next week, so please bare till that time.

Thanks

**Comment by Naresh-ibm** [comment]
@narasimhan-v  this has been taken care in other PR and it got merged. hence closing it.

**Pull request closed without merging**